doc/source/conf.py

```diff
@@ -59,9 +60,9 @@
     # convert rst to md for ipynb
     "pypandoc": True,
     # path to your examples scripts
-    "examples_dirs": ["../../examples/basic"],
+    "examples_dirs": ["../../examples/"],
     # path where to save gallery generated examples
-    "gallery_dirs": ["basic"],
+    "gallery_dirs": ["examples/gallery_examples"],
```

Comment:
I would leave it as examples if possible...

Reply:
Updated "examples/gallery_examples" -> "examples"